start-prod.sh

```diff
@@ -15,5 +15,5 @@ export PYTHONUNBUFFERED=0
 cron
 
 python manage.py migrate
+python manage.py load_socialaccount
```

Comment:
Where the does the manage command for social accts run when testing on travis?
Also what happens if this command is not run? For example, on a dev install? Will app signup still work? Will there be any sort of error that suggests to the developer what the problem is?

Reply:
It doesn't run at all during tests -- we manually create the social accounts in the tests. The only way you would get an error in dev is if you a) manually create or download a socialaccounts_secrets.json file and then b) do not run the management command to load those social accounts into the database. Since a) is a manual process, as is /api/initmodules, I thought it would be reasonable for b) to be manual as well. I can update the wiki.

Reply:
Yes please update the wiki. Do the signup and login screens still work if no secrets are installed?